src/atoms/input/AppInput.ts

```diff
@@ -0,0 +1,43 @@
+import { TInput } from '../../types';
+
+export function AppInput({
+    type = 'text',
+    placeholder = 'Enter your email address...',
```

Comment:
Not a good choise for a default placeholder value, this text is only used once in the design. Better default value would be an empty string.

Reply:
done